catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/DumpCommandTest.java

```diff
@@ -209,54 +204,13 @@ public void testNoFileDumpedWhenTransformFails() throws Exception {
     assertThat(consoleOutput.getOutput(), containsString(" 0 file(s) dumped in "));
   }
 
-  @Test
```

Comment:
I added a unit test for the expected behavior. 

This test was removed because the context of this test was lost since the zip compression class used to be a query response transformer.